top-5-dog-gifs.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: page
+title: Dog Gif Page
+description: Best dogs
+---
+# OpenHack's favorite dog gifs
+
+As selected by @vormwald
+
+## Number 1
+
+![](https://vineyardclubw.blob.core.windows.net/images/dog_mixologist.gif)
```

Comment:
This image doesn't render properly... I'm not sure why, maybe try this one instead? http://blogfiles.wfmu.org/KF/2013/09/25/dog_mixologist.gif